client/state/help/selectors.js

```diff
@@ -1,25 +1,46 @@
 /** @format */
 
 /**
- * Internal dependencies
+ * External dependencies
  */
+import { get } from 'lodash';
 
+/**
+ * Internal dependencies
+ */
 import { getSelectedOrPrimarySiteId } from 'state/selectors';
 import { getSite } from 'state/sites/selectors';
 
 export const getHelpSiteId = state => state.help.selectedSiteId;
 
+/*
+ * Returns the site the customer wishes to request help for. Returns in order of preference:
+ *  - The site the customer has explicitly selected
+ *  - A selected site
+ *  - The primary site (which may not be available).
+ * @param state - global state
+ * @returns {?Object} the help site or null
+ */
 export const getHelpSelectedSite = state => {
 	const siteId = getHelpSiteId( state ) || getSelectedOrPrimarySiteId( state );
-
-	return getSite( state, siteId );
+	const helpSite = getSite( state, siteId );
+	// Are sites loaded but the help site is not available? We may have a bad site or primary.
+	if ( ! helpSite && get( state, 'sites.items.length', 0 ) > 0 ) {
+		const firstSiteId = get( state, [ 'sites', 'items', 0, 'ID' ] );
+		return getSite( state, firstSiteId );
```

Comment:
if we invert this logic it might read easier or at least clear up some redundancy

```js
const helpSite = getSite( state, siteId );

if ( helpSite ) {
	return helpSite;
}

const firstSiteId = get( state, [ 'sites', 'items', 0, 'ID' ] );
if ( ! firstSiteId ) {
	return null;
}

return getSite( state, firstSiteId );
```

Reply:
Let me know how you like the update, items is actually an object here so I had to change it.